File: XWeather/XWeather/Domain/Wu/Webcams.cs
using System.Collections.Generic;

namespace XWeather.Domain
{
	public class Webcams : WuObject
	{
		public override string WuKey => "webcams";

		public List<Webcam> webcams { get; set; }
	}

	public class Webcam
	{
		public string handle { get; set; }
		public string camid { get; set; }
		public string camindex { get; set; }
		public string assoc_station_id { get; set; }
		public string link { get; set; }
		public string linktext { get; set; }
		public string cameratype { get; set; }
		public string organization { get; set; }
		public string neighborhood { get; set; }
		public string zip { get; set; }
		public string city { get; set; }
		public string state { get; set; }
		public string country { get; set; }
		public string tzname { get; set; }
		public string lat { get; set; }
		public string lon { get; set; }
		public string updated { get; set; }
		public string updated_epoch { get; set; }
		public string downloaded { get; set; }
		public string isrecent { get; set; }
		public string CURRENTIMAGEURL { get; set; }
		public string WIDGETCURRENTIMAGEURL { get; set; }
		public string CAMURL { get; set; }
	}
}

File: Swift/XWeather/XWeather/Domain/Wu/Webcams.swift
//
//  Webcams.swift
//  XWeather
//
//  Created by Colby Williams on 9/13/16.
//  Copyright © 2016 Colby Williams. All rights reserved.
//

import Foundation

public class Webcams : WuObject {

	public override var WuKey: String { get { return "webcams" } }

	public var webcams: [Webcam]?
}

public class Webcam {

	public var handle: String?
	public var camid: String?
	public var camindex: String?
	public var assoc_station_id: String?
	public var link: String?
	public var linktext: String?
	public var cameratype: String?
	public var organization: String?
	public var neighborhood: String?
	public var zip: String?
	public var city: String?
	public var state: String?
	public var country: String?
	public var tzname: String?
	public var lat: String?
	public var lon: String?
	public var updated: String?
	public var updated_epoch: String?
	public var downloaded: String?
	public var isrecent: String?
	public var CURRENTIMAGEURL: String?
	public var WIDGETCURRENTIMAGEURL: String?
	public var CAMURL: String?
}
